redirects.yaml

```diff
@@ -251,6 +251,7 @@ contact/whitepaper/?: "https://insights.ubuntu.com/white-papers/"
 containers/docker-ubuntu/?: "/containers"
 containers/kubernetes/?: "/kubernetes"
 containers/lxd/?: "https://linuxcontainers.org/"
+data-privacy/?: "/legal/data-privacy"
```

Comment:
I prefer this URL but the URL is /legal/dataprivacy

Reply:
this clean-up will be in my next pr

Reply:
This URL will 404 if you leave it as it is.